src/coreclr/gc/gc.cpp

```diff
@@ -11641,6 +11661,36 @@ size_t gc_heap::get_segment_size_hard_limit (uint32_t* num_heaps, bool should_ad
     return aligned_seg_size;
 }
 
+#ifdef USE_REGIONS
+bool allocate_initial_regions(int number_of_heaps)
+{
+    initial_regions = new (nothrow) uint8_t*[number_of_heaps][total_generation_count][2];
+    if (initial_regions == nullptr)
+    {
+        return false;
+    }
+    for (int i = 0; i < number_of_heaps; i++)
+    {
+        bool succeed = global_region_allocator.allocate_large_region(&initial_regions[i][poh_generation][0], &initial_regions[i][poh_generation][1], allocate_forward);
```

Comment:
nit - this is a very long line. please keep the lines <= 110 chars. same with the other 2 places.